inst/include/child_model_simulation.hpp

```diff
@@ -349,30 +357,33 @@ void adjust_optAB_bf_transmission_rate(int t,
                                        const State<ModelVariant, real_type> &state_curr,
                                        State<ModelVariant, real_type> &state_next,
                                        IntermediateData<ModelVariant, real_type> &intermediate) {
-  const auto cpars = pars.children.children;
   static_assert(ModelVariant::run_child_model,
                 "adjust_optAB_bf_transmission_rate can only be called for model variants where run_child_model is true");
+  const auto& p_hc = pars.children.children;
+  auto& i_hc = intermediate.children;
 
   internal::calc_wlhiv_cd4_proportion(t, pars, state_curr, state_next, intermediate);
 
-  //Option A and B were only authorized for women with greater than 350 CD4, so if the percentage of women
-  //on option A/B > the proportion of women in this cd4 category, we assume that some must have a cd4 less than 350
-  //option AB will be less effective for these women so we adjust for that
-
-  if (intermediate.children.prop_wlhiv_gte350 > 0) {
-    if ((intermediate.children.PMTCT_coverage(0) + intermediate.children.PMTCT_coverage(1)) > intermediate.children.prop_wlhiv_gte350) {
-      intermediate.children.excessratio_bf = intermediate.children.PMTCT_coverage(0) + intermediate.children.PMTCT_coverage(1) - intermediate.children.prop_wlhiv_gte350;
-      intermediate.children.optA_bf_transmission_rate = (intermediate.children.prop_wlhiv_gte350 * cpars.PMTCT_transmission_rate(4,0,1)) + intermediate.children.excessratio_bf * (1.45 / 0.46) * cpars.PMTCT_transmission_rate(4,0,1) / (intermediate.children.prop_wlhiv_gte350 + intermediate.children.excessratio_bf);
-      intermediate.children.optB_bf_transmission_rate = (intermediate.children.prop_wlhiv_gte350 * cpars.PMTCT_transmission_rate(4,1,1)) + intermediate.children.excessratio_bf * (1.45 / 0.46) * cpars.PMTCT_transmission_rate(4,1,1) / (intermediate.children.prop_wlhiv_gte350 + intermediate.children.excessratio_bf);
+  // Option A and B were only authorized for women with greater than 350 CD4, so if the percentage of women
+  // on option A/B > the proportion of women in this cd4 category, we assume that some must have a cd4 less than 350
+  // option AB will be less effective for these women so we adjust for that
+
+  if (i_hc.prop_wlhiv_gte350 > 0) {
+    auto total_PMTCT_coverage = i_hc.PMTCT_coverage(0) + i_hc.PMTCT_coverage(1);
+    if (total_PMTCT_coverage > i_hc.prop_wlhiv_gte350) {
+      i_hc.excessratio_bf = total_PMTCT_coverage - i_hc.prop_wlhiv_gte350;
+      auto excessfactor_bf = i_hc.excessratio_bf / total_PMTCT_coverage * (1.45 / 0.46) +
+                             i_hc.prop_wlhiv_gte350;
+      i_hc.optA_bf_transmission_rate = excessfactor_bf * p_hc.PMTCT_transmission_rate(4, 0, 1);
+      i_hc.optB_bf_transmission_rate = excessfactor_bf * p_hc.PMTCT_transmission_rate(4, 1, 1);
```

Comment:
lets change these to opt_A_bf_transmission_rate and opt_B_transmission_rate

Reply:
marking for call tomorrow: new ticket

Reply:
for the camel case to snake case pr for me